### DISCUSSION TITLE
Feature request: Heritage

### DISCUSSION THREAD

**Discussion by 501Ghost** [organicmaps/organicmaps#3688] [Ideas]
I have a feature suggestion: **[heritage](https://wiki.openstreetmap.org/wiki/Key:heritage)**

It doesn't have to stand out on the map, Organic Maps isn't [gk.historic.place](http://gk.historic.place/) after all. 

When you click on an object tagged with `heritage=*`, a description in italic could appear that simply says "_heritage_" as default description,  and "_national heritage_" for `heritage=2`. Optionally, "_UNESCO (world) heritage_" for `heritage=1` + `heritage:operator=whc` and similar descriptions for `heritage=4` and `heritage=8` can also be added.

The Wiki documentation states that the value of `heritage=*` should refer to an admin_level, so for example `heritage=2` is national heritage. [Taginfo](https://taginfo.openstreetmap.org/keys/heritage#values) shows a long tail end of dubious values. I think these can be omitted and only the more common values have to be considered for this. `heritage=yes` could IMO still be considered for usage as a generic tag that just means _"heritage_".

It is possible to take this idea a step further and even make hyperlinks to the heritage reference numbers. In that case `heritage=2` + `heritage:operator=rce` + `ref:rce=16722` would be a valid tagging combination that leads to [this page](https://monumentenregister.cultureelerfgoed.nl/monumenten/16722) in the Dutch register of national cultural heritage. I haven't done a market analysis to see if there is a demand for it, I'm just sharing the idea here.

**Comment by biodranik** [comment]
The idea is good, but what is its main use case of it? Who will use it, how, and why?

**Reply by Outlet2048** [reply]
I think it would be cool to see that when traveling. "oh look, this is a unesco world heritage" makes it more interesting.

**Comment by 501Ghost** [comment]
My idea of it is that people who click on an object are at least a bit interested in it. Organic Maps is already sharing other related info about objects, such as accessibility, contact details and related Wikipedia articles. 

A heritage status generally indicates the historical importance of an object. National heritage status is given to objects of national historical importance, for example. So it's mainly to put objects better in a historical context for both tourists and locals who are interested enough to click on an object and read a brief description to learn about it.

**Reply by biodranik** [reply]
What would be the best idea to mark these objects in a way to not distract attention from other important things? Only add a line in the object's place page, when it's selected?

**Reply by 501Ghost** [reply]
Showing a line of text when a heritage object is selected seems good to me. Additionally, UNESCO World Heritage can have the UNESCO logo on the left side of that line, similar to how the logo of Wikimedia Commons is shown. 

It is possible to add other heritage logos like [this one](https://commons.m.wikimedia.org/wiki/File:Beschermd_monument.jpg) too, but for that we would need to collect a bunch of logos of national and sub-national heritage registers. 
Just like with the hyperlinks to the webpages of heritage registers, I would only consider this after the initial release of showing heritage on the map if there is a demand for it.

**Reply by biodranik** [reply]
To move on with this idea, we need to plan and decide:
1. Will icons on the map be somehow different/highlighted for a heritage? (it is more about the future when we support dynamic icons styling)
2. What about the search? Should they be found when people are looking for "sightseeing"? Should they be somehow uniquely highlighted in the search results?
3. How the Place Page will look when it is selected? A small icon under the title so it will be immediately visible? If yes, which one? A line of text/link lower, under coordinates (but then users need to drag/open the Place Page up to see it) with an icon (which one)?
4. Should it be editable by the built-in OM Editor?

**Reply by 501Ghost** [reply]
I have no experience in app development, so I am sharing my own ideas and preferences. Feel free to disagree with or object to what I'm saying here.

1. 
Map icons and perhaps also buildings that are now grey or brown can be displayed more brightly to make them stand out a bit more. I suggest a bright, almost shiny but not too saturated bronze colour that resembles the brown colour which is often used for historic objects.

For UNESCO world heritage a [separate icon](https://commons.m.wikimedia.org/wiki/File:Unesco_World_Heritage_logo_notext_transparent.svg) can be considered, but other map icons may not be memorable or recognisable for most users and we should remember the KISS principle. 

2. 
> Should they be found when people are looking for "sightseeing"?

I would say so, at least for named objects. Heritage and sightseeing go hand in hand for many tourists, at least in Europe.

> Should they be somehow uniquely highlighted in the search results?

I don't know and I don't have an opinion on this.

3. 
AFAIK there is no universal "heritage" icon (please let me know if I'm wrong), so instead of a small icon I suggest the text "UNESCO World Heritage_", "_National Heritage_", or simply "_Heritage_" under the title. The text for "UNESCO World Heritage" can be paired with an icon in the same way as Wikimedia Commons.

So when you click on [Fort Vechten](https://www.openstreetmap.org/way/623200845) on the map, you would see:
* **Fort Vechten**
* Fort
* National heritage _(this would be a new line)_

To continue with this example, when you swipe to the full information (where contact details and Wiki links are displayed), there can be an extra line that refers to the Dutch national heritage register including a hyperlink to [page 531252](https://monumentenregister.cultureelerfgoed.nl/monumenten/531252). We would have to implement this for UNESCO heritage and for national registers that are documented on the Key:heritage page on the OSM Wiki and which are actually in use.

I considered similar descriptions for objects in sub-national heritage registers, but for that we would need to apply appropriate names and descriptions for these registers. The main issue is that this differs per country, so we risk calling for example `heritage=4` in the USA "_Provincial heritage_" instead of "_State heritage_". This can be fixed for a few countries but probably not for all. For simplicity I would just call all sub-national heritage "_Heritage_" and treat it as `heritage=yes`, still omitting the dubious and rare `heritage=*` tag values.

4. 
This would be useful in places where heritage is recognisable on the ground. In NL there are often heritage logos on structures to inform viewers of the structure's heritage status. This would be easy to do for most users. I don't know if the situation is similar in other countries. On the other hand, if mappers start mapping any historic object with `heritage=yes`, this could become counterproductive. There are advantages and disadvantages to this, and I can't make this decision.

**Comment by 501Ghost** [comment]
P.s. This is especially true for `historic=*` objects, as the historical importance doesn't become clear from the `historic=*` tag itself, as highlighted in [this long discussion on the Community forum](https://community.openstreetmap.org/t/feature-proposal-rfc-historic/3910/).
